### PR TITLE
Feat- Added Rules page

### PR DESCRIPTION
**Pull request by SukanyaSen530**
Added rules page.

### CODE REVIEW

File: pages/rules.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  list-style-type: circle;
+  list-style-position: inside;
+}
+
+.rules-points > li {
+  margin: 1rem 0;
+}
+
+.rules-btn {
+  margin-top: 2rem;
```
Comment (AmishMishra11) [inline pages/rules.css:25]: you could add css variables as you are using same value in many places

**Review by AmishMishra11 — COMMENTED** [review]
Looks amazing, Great job

File: pages/rules.css (hunk 1 above)
Comment (SukanyaSen530) [reply pages/rules.css:25]: Willl refactor at last :) thanks for the suggestion.

**Review by sharathnair9999 — APPROVED** [review]
The UI and the animation of landing page looks nice. Good work

**Pull request merged**